tests/unit/members-dao.test.js

```diff
@@ -0,0 +1,300 @@
+const appRoot = require('app-root-path');
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const chaiExclude = require('chai-exclude');
+const _ = require('lodash');
+const sinon = require('sinon');
+const proxyquire = require('proxyquire');
+const { createConnStub } = require('./test-util');
+const {
+  fakeId,
+  fakeMemberPostBody,
+  fakeEmail,
+  fakeName,
+  fakeMemberPatchBody,
+  testCases,
+} = require('./test-data');
+
+// sinon.replace(config, 'get', () => ({ oracledb: {} }));
```

Comment:
Seems like this line is only needed in one test class. Remove the comment if it is not needed.